.gitignore

```diff
@@ -1,66 +0,0 @@
-# Logs
```

Comment:
Nah, you should not delete this file, you should revet it to the original one in the main branch https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore

Reply:
Well done👏